lisa/util/constants.py

```diff
@@ -136,3 +136,6 @@
 DEVICE_TYPE_GPU = "GPU"
 
 DISK_PERFORMANCE_TOOL = "fio"
+
+# special variables
+SPECIAL_VARIABLE_RUNBOOK_DIR = "runbook_dir"
```

Comment:
1. the constants.RUNBOOK_PATH is the runbook's path.
2. I prefer not to create hidden variable, because it breaks the readability of the runbook.

Reply:
Oh, I was looking for `RUNBOOK_PATH` in the `Environment` object. I wasn't expecting it to be a global variable.